installer_windows_local.py

```
@@ -613,11 +641,21 @@ def run_installation(self):
-            # Always ensure frontend is ready, even with --skip-install
-            if not self.install_frontend_deps():
-                self.logger.warning("Frontend dependency installation failed.")
-            if not self.build_frontend():
-                self.logger.warning("Frontend build failed.")
+            # Always ensure frontend is ready, even with --skip-install.
+            # Wrapped in try/except so any unexpected exception here does NOT
+            # kill the entire installation (Python backend is already installed).
+            try:
+                if not self.install_frontend_deps():
+                    self.logger.warning("Frontend dependency installation failed.")
+            except Exception as fe:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)